src/components/Notification/Notification.tsx

```diff
@@ -22,7 +22,10 @@ interface BaseProps {
   notificationType?: NotificationTypes;
 }
 
-export interface NotificationProps extends BaseProps {
+type BasePropsWithoutAlignText = Omit<BaseProps, "alignText">;
```

Comment:
If the intent of this update is to remove the `alignText` prop, why is this or any other reference to `alignText` still in here?

Reply:
The actual `Notification` component doesn't use `alignText` but the `NotificationContent` component does use it. So it's only for internal use.

Reply:
Got it.  Thanks.